profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -546,63 +650,91 @@ function _nexteuropa_varnish_get_node_paths($node) {
     else {
       $lgregex = '';
     }
-    $original_path = $entity_uri['path'];
-    $options = $entity_uri['options'];
 
     // Get the languages the node is translated into (entity translation).
     if (module_exists('entity_translation') && entity_translation_enabled('node', $node)) {
       $handler = entity_translation_get_handler('node', $node);
       $translations = $handler->getTranslations();
       $lang_codes = array_keys($translations->data);
       $all_languages = entity_translation_languages('node', $node);
-      foreach ($lang_codes as $lang_code) {
-        if (isset($all_languages[$lang_code])) {
-          $alias = drupal_get_path_alias($original_path, $lang_code);
+      // Small workaround for undefined language.
+      if (count($lang_codes) == '1' && 'und' == $lang_codes[0]) {
```

Comment:
use `===`

Reply:
`if (count($lang_codes) === 1 && 'und' === $lang_codes[0])`

Count returns a int, so, don't worry for this one. and I think (to be verified) that 'und' could be replaced with a constant ([LANGUAGE_NONE](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/constant/LANGUAGE_NONE/7.x)), so this is a string and triple equals can be used without any troubles.

Reply:
Makes sense for count, the other is again to risky